<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>analyzeDialog</class>
 <widget class="QDialog" name="analyzeDialog">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>670</width>
    <height>760</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Analiz Sonucu</string>
  </property>
  <property name="windowIcon">
   <iconset resource="icons.qrc">
    <normaloff>:/resources/Weather-icon.png</normaloff>:/resources/Weather-icon.png</iconset>
  </property>
  <widget class="QLabel" name="labelMono">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>320</width>
     <height>320</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="labelAnalyze">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>10</y>
     <width>320</width>
     <height>320</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="plainTextEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>340</y>
     <width>651</width>
     <height>331</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 195);</string>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>680</y>
     <width>651</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 0, 0);</string>
   </property>
   <property name="text">
    <string>ONAYLIYOR MUSUNUZ ?</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="acceptButton">
   <property name="geometry">
    <rect>
     <x>213</x>
     <y>710</y>
     <width>100</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>EVET</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>353</x>
     <y>710</y>
     <width>100</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>HAYIR</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelButton_2">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>710</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>Ã–RNEK</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>acceptButton</sender>
   <signal>clicked()</signal>
   <receiver>analyzeDialog</receiver>
   <slot>acceptButton()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>258</x>
     <y>472</y>
    </hint>
    <hint type="destinationlabel">
     <x>191</x>
     <y>464</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>analyzeDialog</receiver>
   <slot>cancelButton()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>436</x>
     <y>471</y>
    </hint>
    <hint type="destinationlabel">
     <x>467</x>
     <y>466</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancelButton_2</sender>
   <signal>clicked()</signal>
   <receiver>analyzeDialog</receiver>
   <slot>sampleButton()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>596</x>
     <y>468</y>
    </hint>
    <hint type="destinationlabel">
     <x>572</x>
     <y>469</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>acceptButton()</slot>
  <slot>cancelButton()</slot>
  <slot>sampleButton()</slot>
 </slots>
</ui>
